src/main/java/org/fogbowcloud/sebal/parsers/WeatherStation.java

```diff
@@ -140,136 +83,119 @@ protected String readFullRecord(Date date, List<JSONObject> stations, int number
 			LOGGER.info("Stations list is empty");
 		}
 
-
 		return null;
 	}
 
```

Comment:
I cannot understand what this method do by reading its name

Reply:
Well, this method select one station of all stations finded by the _StationOperator_. Do you have any suggestion for this method name?